migrations/2023-02-07-070512_change_merchant_connector_id_data_type/down.sql

```diff
@@ -0,0 +1,10 @@
+CREATE SEQUENCE IF NOT EXISTS merchant_connector_id_seq OWNED BY merchant_connector_account.merchant_connector_id;
+
+UPDATE merchant_connector_account
+SET merchant_connector_id = id;
+
+ALTER TABLE merchant_connector_account
+ALTER COLUMN merchant_connector_id TYPE INTEGER USING (trim(merchant_connector_id)::integer);
+
+ALTER TABLE merchant_connector_account 
+ALTER COLUMN merchant_connector_id SET DEFAULT nextval('merchant_connector_id_seq');
```

Comment:
So here since Serial is not a concrete type we can't use it in ALTER TABLE. So creating Sequence will emulate that behaviour.